### ISSUE TITLE
OS_CODE documentation correction

### ISSUE THREAD

**Issue by ianhays** [madler/zlib#121]
The documentation for using Zlib to write a GZip header seems to be incorrect on the behavior of how the header fields are set.

The docs: ```If deflateSetHeader is not used, the default gzip header has text false, the time set to zero, and os set to 255, with no extra, name, or comment fields.```

However, in update 1.2.2.2 (30 December 2004) this behavior was changed to use OS_CODE in the deflate() default gzip header.

On my Windows machine this results in zlib setting the gzip header to have the OS field set to 0 rather than 255.

**Comment by madler** [comment]
I don't know of anything that looks at that byte, but anyway, please let me know if `_WIN32` is #defined in your compiler.

**Comment by ianhays** [comment]
>please let me know if _WIN32 is #defined in your compiler.

It is defined.

**Comment by madler** [comment]
Then it should be setting it to 11. Are you sure that you're looking at the right byte?

**Comment by ianhays** [comment]
I just checked again and it is is set to 11 now. It's possible I was looking at the wrong byte when I made the original post back in December, though in either case the docs are misleading.

>I don't know of anything that looks at that byte,

We don't use the byte for anything either; we just wanted to make sure we were properly delegating to zlib to let it decide how to set the byte. The docs led me to believe we were doing this incorrectly, as It was not being set to FF as they said it should be (since we don't use `deflateSetHeader`).
